### ISSUE TITLE
sync ~/.ssh/config overwrites file in place

### ISSUE THREAD

**Issue by jhit** [muriloventuroso/easyssh#63]
Hi.

I have an extensive ssh config in place but when I start EasySSH the first time it deletes all entries when activate the sync.

Either import the available sites what would be the best. Or at least make a backup copy of the file and inform the user. Otherwise, this feature can cause a lot of trouble.

**Comment by andriusign** [comment]
Confirmed here on two machines. This is quite an embarrassment.

**Comment by apapsch** [comment]
Yeah, the program just trashed my config. Luckily I have a backup.

**Comment by andriusign** [comment]
Not sure why it is closed. Was it solved already? I try running easyssh from terminal to get more insights, however when I go to Preferences and toggle checkboxes program freezes and this error appears:
```
$ flatpak run com.github.muriloventuroso.easyssh

** (com.github.muriloventuroso.easyssh:2): CRITICAL **: 16:44:52.632: file /run/build/easyssh/src/MainWindow.vala: line 237: uncaught error: Failed to execute child process “xdg-mime” (No such file or directory) (g-exec-error-quark, 8)
```

**Comment by muriloventuroso** [comment]
Please use the repository version, not yet released on Flathub.

Em ter, 22 de out de 2019 18:21, andriusign <notifications@github.com>
escreveu:

**Comment by vpArth** [comment]
Now I have two empty files: `~/.ssh/config` and `~/.ssh/config.backup`.
Was it planned to silently replace the backup with a copy of an empty file?
Why not request confirmation from the user?

**Comment by stellarpower** [comment]
I'd say the wording "sync" is ambiguous, at least in English translations. Either "Merge EasySSH config with ~/.ssh/config" if that can be performed, or "overwrite" would be clearer. Or perhaps an option for loading configurations one way but not the other:
- "Load ~/.ssh/config hosts into EasySSH"
- "Save EasySSH hosts into ~/.ssh/config"

**Comment by BadCo-NZ** [comment]
@muriloventuroso any update on this?

I just tried to setup symlinks on my nextcloud instance to use the same file between two machines, but obviously as above, the file keeps getting wiped.

**Comment by Firecul** [comment]
Sync backed up my config before wiping it but it'd still be nice if easyssh could import connections from my current config files
